apps/examples/wifi_manager/wm_test/wm_test_event_tc.c

```diff
@@ -0,0 +1,164 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License\n");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+
+#include <unistd.h>
+#include <string.h>
+#include <queue.h>
+#include <semaphore.h>
+#include <sys/types.h>
+#include <stdio.h>
+#include <errno.h>
+#include <tinyara/net/if/wifi.h>
+#include <wifi_manager/wifi_manager.h>
+#include <stress_tool/st_perf.h>
+#include "wm_test.h"
+#include "wm_test_utils.h"
+#include "wm_test_network.h"
+#include "wm_test_mock.h"
+#include "wm_test_log.h"
+
+#define TAG "[WE]"
+/*
+ * callbacks
+ */
+static void wm_cb_sta_connected(wifi_manager_cb_msg_s msg, void *arg);
+static void wm_cb_sta_disconnected(wifi_manager_cb_msg_s msg, void *arg);
+static void wm_cb_softap_sta_join(wifi_manager_cb_msg_s msg, void *arg);
+static void wm_cb_softap_sta_leave(wifi_manager_cb_msg_s msg, void *arg);
+static void wm_cb_scan_done(wifi_manager_cb_msg_s msg, void *arg);
+
+static wifi_manager_cb_s g_wifi_callbacks = {
+	wm_cb_sta_connected,
+	wm_cb_sta_disconnected,
+	wm_cb_softap_sta_join,
```

Comment:
wm_cb_softap_sta_joined and wm_cb_softap_sta_left might be better for consistency in naming.

Reply:
Thanks I'll modify as your recommendation.